test/Npgsql.Tests/SecurityTests.cs

```diff
@@ -162,6 +178,36 @@ public void Bug1718()
             }
         }
 
+        [Test]
+        [Timeout(2000)]
+        public void ConnectToDatabaseUsingScramPlus() 
+        {
+            var csb = new NpgsqlConnectionStringBuilder(ConnectionString)
+            {
+                SslMode = SslMode.Require,
+                TrustServerCertificate = true,
+                Username = "npgsql_tests_scram",
+                Password = "npgsql_tests_scram",
+            };
+
+            try
+            {
+                using var conn = OpenConnection(csb);
+                // scram-sha-256-plus only works begining from PostgreSQL 11
+                if (conn.PostgreSqlVersion.Major >= 11)
+                {
+                    Assert.That(conn.IsScramPlus, Is.True);
+                }
+            }
+            catch (Exception e)
+            {
+                if (TestUtil.IsOnBuildServer)
+                    throw;
```

Comment:
nit: use exception filter:

```suggestion
            catch (Exception e) when (!TestUtil.IsOnBuildServer)
            {
```